plugins/modules/skydive_capture.py

```diff
@@ -86,9 +88,9 @@
     - absent
 """
 
-EXAMPLES = """
+EXAMPLES = r"""
 - name: start a new flow capture directly from gremlin query
-  skydive_capture:
+  community.skydive.skydive_capture:
```

Comment:
@gundalow Is it required ?

Reply:
Examples should use FQCN, do this change is correct.

An example of this being done in an other collection: https://github.com/ansible-collections/community.general/pull/643/files